removed all instances of the var "funfact" and replaced w/ "bio". mig…

ktakattack opened the pull request:
…rated to DB.

Do not merge until Eyitayo reviews since he's the DB expert.

tayoogunbiyi commented:
I’m no expert haha. Would review this.

tayoogunbiyi commented on models/user.js:
```diff
@@ -1,29 +1,28 @@
+      // define association here
+    }
+  };
+  User.init({
+    username: DataTypes.STRING,
+    discordId: DataTypes.STRING,
+    linkedin: DataTypes.STRING,
+    github: DataTypes.STRING,
+    bio: DataTypes.STRING,
```
This file is auto-generated so we don't want to manually edit it. We could, but however, the migration file should definitely not be auto edited. We should have a new migration file that reflects this specific DB structure change.

tayoogunbiyi commented:
@ktakattack I added a new migration file so the MySQL knows about the column change. then I changed a place where the fun fact was still being referenced.

LGMT!

ktakattack commented:
> @ktakattack I added a new migration file so the MySQL knows about the column change. then I changed a place where the fun fact was still being referenced.
> 
> LGTM!

Thank you for the assist!
